main.go

```diff
@@ -152,18 +173,43 @@ func main() {
 		}
 		duration = time.Second * time.Duration(RateLimitSpecs.PollInterval)
 	}
+
+	refreshInterval := int64(duration.Seconds())
+
+	buildGraph()
+	GetRateLimits(token)
+	buildDashboard(refreshInterval)
+
+	secondsToWait := refreshInterval
+	lastUpdated := GetLastUpdateTime()
+
 	for {
-		secondsToWait := duration.Seconds()
 		for {
-			if secondsToWait <= 0 {
+			if secondsToWait == 0 {
```

Comment:
Is there absolutely no chance for the code to desync and make `secondsToWait` less than 0?